src/pages/documentation/resources/index.md

```diff
@@ -20,7 +20,15 @@ To see the most frequently asked questions about Mobile SDK, please read the doc
 
 You can migrate your Objective-C-based version of the SDK to the Swift-based version of the SDK. 
 
-To learn know how to migrate your Mobile SDK implementation to Swift, please read the guide on [migrating to Swift](../migrate-to-swift.md).
+To learn how to migrate your Mobile SDK implementation to Swift, please read the guide on [migrating to Swift](../migrate-to-swift.md).
+
+## Managing Gradle dependencies 
+
+To learn how to manage Gradle dependencies of Mobile SDKs, please read the guide on [managing Gradle dependencies](../manage-gradle-dependencies.md).
+
+## Managing SPM dependencies 
+
+To learn how to manage SPM dependencies of Mobile SDKs, please read the guide on [managing SPM dependencies](../manage-spm-dependencies.md).
```

Comment:
Should we surface this better under resources in AEP docs repo.
```
## Integrating SDKs in mobile apps

To learn how to integrate the SDKs in your mobile apps, please read the [manage Gradle dependencies](../manage-gradle-dependencies.md) guide for Android and [manage SPM dependencies](../manage-spm-dependencies.md) guide for iOS.
```

Reply:
I will let Caleb structure the content, as you mentioned in your previous comment. We will also link to the "manage Gradle dependencies" page from Launch install instructions window.

Reply:
@calebk1 ⏫ Please make a note.

Reply:
Yeah, I'll be adding it to the install instructions once this gets merged in as well.